packit_service/service/static/style.css

```diff
@@ -2,7 +2,11 @@
   font-family: sans-serif;
 }
 
-pre {
+pre,
+code,
+kbd,
+var,
+samp {
```

Comment:
what are these? :D

Reply:
answered myself: https://stackoverflow.com/questions/32284477/what-are-the-differences-between-kbd-samp-and-code-in-html